src/EMS/PatientFields.js

```diff
@@ -1,15 +1,62 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 
 import FormCheckbox from '../Components/FormCheckbox';
+import FormComboBox from '../Components/FormComboBox';
 import FormInput from '../Components/FormInput';
 import FormRadio from '../Components/FormRadio';
 import FormRadioFieldSet from '../Components/FormRadioFieldSet';
 import FormTextArea from '../Components/FormTextArea';
 import Heading from '../Components/Heading';
+
 import Ringdown from '../Models/Ringdown';
+import ApiService from '../ApiService';
 
 function PatientFields({ ringdown, onChange }) {
+  const [ambulanceId, setAmbulanceId] = useState([]);
+  const [dispatchCall, setDispatchCall] = useState([]);
```

Comment:
These names, ambulanceId and dispatchCall, are misleading- they sound like singleton objects, but they actually store arrays, a _collection_ of objects.

Reply:
I've renamed these to be plural